src/site/_data/authorsData.json

```diff
@@ -1569,5 +1569,11 @@
     "linkedin": "https://www.linkedin.com/in/jimmymooney/",
     "country": "US",
     "image": "image/Xyvh8LLq27V5yRjH5iS1dbf64pE2/M2E7ke6rO6ZpIabYeizX.jpg"
+  },
+  "alexeyrodionov": {
+    "country": "RU",
+    "twitter": "alexey_rodionov",
+    "github": "FluorescentHallucinogen",
+    "image": "image/Xyvh8LLq27V5yRjH5iS1dbf64pE2/M2E7ke6rO6ZpIabYeizZ.png"
```

Comment:
The image URL will be updated.

Reply:
Final URL `image/8WbTDNrhLsU0El80frMBGE4eMCD3/n6mW0LtLb2NtYAoQSCx7.png`.

Reply:
Changed.